__tests__/app.test.js

```diff
@@ -375,3 +375,42 @@ describe("GET api/users", () => {
       });
   });
 });
+
+describe("QUERY TOPIC api/articles", () => {
+  test("GET:200 returns a filtered array of article objects with corresponding properties", () => {
+    return request(app)
+      .get("/api/articles?topic=cats")
+      .expect(200)
+      .then((response) => {
+        const articles = response.body.articles;
+        expect(articles.length).toBe(1);
+        expect(articles[0]).toMatchObject({
+          article_id: 5,
+          title: "UNCOVERED: catspiracy to bring down democracy",
+          topic: "cats",
+          author: "rogersop",
+          created_at: expect.any(String),
+          votes: 0,
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          comment_count: "2",
+        });
+      });
+  });
+  test("GET:200 returns an error message when no articles of queried topic", () => {
+    return request(app)
+      .get("/api/articles?topic=TEST")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.msg).toBe("no articles of this topic");
```

Comment:
In the event that you're trying to filter by a topic that doesn't exist in the database, this would be a `404`. If the topic does exist but has no articles attached, we would expect an _empty array_ to be sent